v-next/hardhat/src/internal/core/plugins/detect-plugin-npm-dependency-problems.ts

```diff
@@ -78,7 +78,7 @@ export async function detectPluginNpmDependencyProblems(
 
     const { satisfies } = await import("semver");
 
-    if (!satisfies(installedVersion, versionSpec)) {
+    if (!satisfies(installedVersion, versionSpec.replace("workspace:", ""))) {
```

Comment:
Satisfies failed on the version spec having the network prefix.

Reply:
good catch